src/main/resources/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator/github-connector.css

```diff
@@ -0,0 +1,79 @@
+.create-github-api-key-container {
```

Comment:
Sorry, I guess... hopefully all this CSS could just go away if we had access to Bootstrap.